ui/conf/nginx.conf

```diff
@@ -15,5 +15,9 @@ server {
 
         try_files /override-brand/$1 /brand/$1;
     }
+
+    location /config.json {
+        alias /etc/metalk8s-ui/config.json;
```

Comment:
We already have `/etc/metalk8s`, let's use something like `/etc/metalk8s/ui/config.json` or similar? Unless `metalk8s-ui` were to become a separate (RPM) package at some point. WDYT, @scality/metalk8s ?

Reply:
How about ```/usr/share/nginx/html```?